lib/emitter.js

```diff
@@ -1,13 +1,14 @@
 var
-	_         = require('lodash'),
-	assert    = require('assert'),
-	events    = require('events'),
-	net       = require('net'),
-	os        = require('os'),
 	stream    = require('readable-stream'),
+	discard   = require('discard-stream'),
 	UDPStream = require('./udp-stream'),
+	_         = require('lodash'),
+	events    = require('events'),
+	assert    = require('assert'),
+	util      = require('util'),
 	url       = require('url'),
-	util      = require('util')
+	net       = require('net'),
+	os        = require('os')
```

Comment:
(Sorry about the reordering here — I did this with a macro, the only new `require` is `discard-stream`. I can undo it if you'd like.)